packages/controller-utils/src/constants.ts

```diff
@@ -8,6 +8,7 @@ import {
 export const RPC = 'rpc';
 export const FALL_BACK_VS_CURRENCY = 'ETH';
 export const IPFS_DEFAULT_GATEWAY_URL = 'https://cloudflare-ipfs.com/ipfs/';
+export const NFT_UPDATE_THRESHOLD = 500;
```

Comment:
I have randomly chosen 500; might still be too much; we can probably lower it

Reply:
It looks like this change is being added to `controller-utils`, which is designed to be shared among all controller packages. Is that right, do you expect other controller packages to need this constant? If not, should this only go inside of `assets-controllers`? (I know that there are other NFT-related things here, but I have a similar question toward those.)

Reply:
@sahar-fehri any thoughts on this ?

Reply:
I do not expect this to be used by other controllers; i can remove the NFT_UPDATE_THRESHOLD to be part of NftController.ts 🙏